packages/evolution-frontend/src/components/survey/Question.tsx

```diff
@@ -329,9 +329,9 @@ export class Question<CustomSurvey, CustomHousehold, CustomHome, CustomPerson> e
         };
         const content = [(
             <div
-                key={"content_" + this.props.path}
+                key={'content_' + this.props.path}
                 style={{ position: 'relative' }}
-                className={`apptr__form-container${twoColumns ? ' two-columns' : ''}${
+                className={`apptr__form-container apptr__form-join-${this.props.join || 'none'}${twoColumns ? ' two-columns' : ''}${
```

Comment:
Is there a use case for the style `apptr__form-join-none`? Why not just not add a style if there is no join?

Also, now that we don't push/unshift an element to the content array, the content does not need to be an array anymore.

Reply:
Where is the push/unshift in the code?

Reply:
You removed it